pkg/cli/cliflags/flags.go

```diff
@@ -293,6 +293,14 @@ csv, table, records, sql, raw, html. If left unspecified, defaults to
 tsv for non-interactive sessions and table for interactive sessions.`,
 	}
 
+	ClusterName = FlagInfo{
+		Name: "cluster-name",
+		Description: `
+Sets a name to verify the identity of a remote node or cluster, when
+either the node or cluster, or both, have not yet been initialized
+yet.`,
```

Comment:
Could you mention that all nodes must have the same cluster name if set.

Reply:
Done.